lib/watcher/status-wd.js

```diff
@@ -9,6 +10,7 @@ exports.worker = function(handle) {
       cwd: process.cwd(),
       pid: process.pid,
       isTracing: !!tracer(),
+      applicationName: agent().config.appName,
```

Comment:
I will remove `appName` from the status message since it doesn't make sense there anymore.
`applicationName` is consistent with the instance model in s-m-m.